apps/sms/views/conversation/js/conversation.js

```diff
@@ -80,6 +80,11 @@ var ConversationView = {
   showErrorInFailedEvent: '',
   previousSegment: 0,
 
+  /**
+   * Set to true after init is run.
+   */
+  _isReady: false,
```

Comment:
__question:__ wondering if we can use approach we used in MessageManager, but with ```init``` inside [1]. I don't say it's good or bad, just throwing out ideas.

[1] https://github.com/mozilla-b2g/gaia/blob/14e32276025b0310d3e89027320cf4b2a24cedfb/apps/sms/services/js/message_manager.js#L541